services/docker-compose.yml

```diff
@@ -755,9 +755,10 @@ services:
 
       # WEBSERVER_PROJECTS
       WEBSERVER_PROJECTS: ${WEBSERVER_PROJECTS}
+      PROJECTS_INACTIVITY_INTERVAL: ${PROJECTS_INACTIVITY_INTERVAL}
       PROJECTS_MAX_COPY_SIZE_BYTES: ${PROJECTS_MAX_COPY_SIZE_BYTES}
       PROJECTS_MAX_NUM_RUNNING_DYNAMIC_NODES: ${PROJECTS_MAX_NUM_RUNNING_DYNAMIC_NODES}
-      PROJECTS_INACTIVITY_INTERVAL: ${PROJECTS_INACTIVITY_INTERVAL}
+      PROJECTS_TRASH_RETENTION_DAYS: ${PROJECTS_TRASH_RETENTION_DAYS}
```

Comment:
you will need some devops changes as well

Reply:
Exacly. see link in the description to that PR.